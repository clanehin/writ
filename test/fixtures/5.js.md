```js
//:: Spaces are allowed in keys :://
```

```js
//== Spaces are allowed in keys ==//

"isn't it neat that spaces are allowed in keys?";
```
