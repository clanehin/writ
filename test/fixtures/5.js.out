"isn't it neat that spaces are allowed in keys?";
